Microsoft.Research/Contracts/MsCorlib/System.Runtime.CompilerServices.ExtensionAttribute.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace System.Runtime.CompilerServices
 {
+#if NETFRAMEWORK_4_5
```

Comment:
It is like this everywhere else, I just wanted to comply to the existing pattern.
With this both ExtensionAttribute definitions look the same.
If we want to change this, we should do a bulk refactoring.